tests/src/test/scala/akka/kafka/TransactionsOps.scala

```diff
@@ -163,7 +175,6 @@ trait TransactionsOps extends TestSuite with Matchers {
       .plainSource(settings, Subscriptions.topics(topic))
       .map(r => (r.partition(), r.offset(), r.value()))
       .take(elementsToTake)
-      .idleTimeout(30.seconds)
```

Comment:
Prevent prematurely timing out the stream when consumer is idle.  Give `maxRestarts` a chance to elapse and then all remaining elements will be consumed.